Dockerfile

```diff
@@ -0,0 +1,52 @@
+# syntax=docker/dockerfile:1
+
+ARG GO_VERSION=1.19.2
+ARG XX_VERSION=1.1.2
+ARG OSXCROSS_VERSION=12.3-r0-alpine
+
+# xx is a helper for cross-compilation
+FROM --platform=$BUILDPLATFORM tonistiigi/xx:${XX_VERSION} AS xx
+
+# osxcross contains the MacOSX cross toolchain for xx
+FROM crazymax/osxcross:${OSXCROSS_VERSION} AS osxcross
+
+FROM --platform=$BUILDPLATFORM golang:${GO_VERSION}-alpine3.16 AS base
+COPY --link --from=xx / /
+RUN apk add --no-cache clang git lld llvm make
+ARG TARGETPLATFORM
+WORKDIR /src
+COPY --link src/cmd/linuxkit/ .
```

Comment:
Remove this line, it's better to use `RUN` mount, see comment about single stage.

Reply:
:-) I was exactly working on that